data/src/main/java/io/gnosis/data/repositories/TransactionRepository.kt

```diff
@@ -121,14 +121,19 @@ class TransactionRepository(
         )
     }
 
-    private fun transferErc721(transaction: MultisigTransactionDto): Transaction.Transfer =
-        Transaction.Transfer(
-            transaction.to,
-            transaction.safe,
-            transaction.value,
+    private fun transferErc721(transaction: MultisigTransactionDto): Transaction.Transfer {
+        val from = transaction.dataDecoded?.parameters?.getValueByName("from")?.asEthereumAddress() ?: Solidity.Address(BigInteger.ZERO)
```

Comment:
when is it possible for erc721 transfer to have `from` or `to` values set to zero?

Reply:
I will ask Uxio if that is possible.

Reply:
Uxio said: "No, but it could be possible that there's no data decoded due to this https://vessenes.com/the-erc20-short-address-attack-explained/#:~:text=What%20happens%20in%20this%20attack,given%20to%20the%20shortened%20address.&text=When%20the%20parser%20is%20getting,and%20calls%20it%20a%20day."

Reply:
If there is no data decoded section, then the app will treat it as a custom transaction

Reply:
So I guess we're good here?